src/core/commandVersionUpgrades/upgradeV1ToV2/upgradeV1ToV2.ts

```diff
@@ -131,10 +131,12 @@ function upgradePrimitiveTarget(
         if (modifier?.type === "subpiece") {
           break;
         }
+      // fallthrough
       case "line":
         if (mark?.type === "lineNumber") {
           break;
         }
+      // fallthrough
```

Comment:
I presume these fallthroughs are deliberate?

Reply:
I believe so. Slightly confusing way of doing this logic imo but prob ok as it's just in migration

Cc/ @AndreasArvidsson